packages/osl-swf-devmode-image/generated/logic-swf-devmode-rhel8-image.yaml

```diff
@@ -19,14 +19,14 @@
 schema_version: 1
 name: "openshift-serverless-1/logic-swf-devmode-rhel8"
 from: "registry.access.redhat.com/ubi8/openjdk-17:latest"
-version: "main"
+version: "1.36.0"
 description: "Red Hat build of Kogito Serverless Workflow development mode image with Quarkus extensions libraries preinstalled"
 
 labels:
   - name: "io.quarkus.platform.version"
-    value: "3.15.3"
+    value: "3.15.4.redhat-00001"
```

Comment:
@saumya1singh you can change this value here instead: https://github.com/kubesmarts/kie-tools/blob/main/packages/root-env/env/index.js#L72

Then pnpm bootstrap command should update everything.